templates/common/baremetal/files/baremetal-non-virtual-ip.yaml

```diff
@@ -77,7 +77,7 @@ contents:
 
         @classmethod
         def from_line(cls: Type[TR], line: str) -> TR:
-            items = line.split()
+            items = [i for i in line.split() if i != '\\']
```

Comment:
Aren't route entries always a single line? Did you encounter a `\\` ? I only ever encountered it in Address processing.

Reply:
Apparently they aren't. I had some masters with a route that looks like: "default proto ra metric 100 \   nexthop via fe80::69e0:8df7:4351:4eb6 dev enp1s0 weight 1 \     nexthop via fe80::f620:b5b3:612a:8446 dev enp1s0 weight 1 \     nexthop via fe80::b449:f968:42ce:eef5 dev enp1s0 weight 1 pref medium"